src/ontology/mondo-edit.obo

```diff
@@ -93328,9 +93331,11 @@ xref: ICD10CM:L20.9 {source="DOID:3310"}
 xref: icd11.foundation:215767047 {source="MONDO:equivalentTo"}
 xref: ICD9:691 {source="EFO:0000274"}
 xref: ICD9:691.8 {source="EFO:0000274"}
+xref: MEDGEN:350353 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MEDGEN:41502 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: MESH:D003876 {source="DOID:3310", source="EFO:0000274"}
+xref: MESH:D003876 {source="https://orcid.org/0000-0003-1967-3726", source="DOID:3310", source="EFO:0000274", source="MONDO:equivalentTo"}
 xref: NCIT:C3001 {source="MONDO:equivalentTo"}
+xref: OMIM:603165 {source="MONDO:equivalentTo"}
 xref: OMIMPS:603165 {source="DOID:3310", source="MONDO:equivalentTo"}
```

Comment:
this OMIMPS should be moved over to MONDO:0100178
dermatitis, atopic, susceptibility to

and add 'has characteristic' some inherited to MONDO:0100178

Reply:
@nicolevasilevsky what should be the source for `'has characteristic' some inherited to MONDO:0100178`? An inheritance mechanism is only listed for one of the entries in https://www.omim.org/phenotypicSeries/PS603165 so I am not sure if OMIMPS:603165 is the correct source for the `has characteristic` axiom.